crates/sui-graphql-rpc/src/types/suins_registration.rs

```diff
@@ -47,6 +47,13 @@ pub(crate) struct NameService;
 #[derive(Debug)]
 pub(crate) struct Domain(NativeDomain);
 
+#[derive(Enum, Copy, Clone, Eq, PartialEq)]
+#[graphql(remote = "sui_json_rpc::name_service::DomainFormat")]
```

Comment:
TIL -- what does the `remote` attribute do? (Is that what allows you to write the `into` in `owner.rs`?)

Reply:
Correct, a great suggestion from @wlmyng 
https://async-graphql.github.io/async-graphql/en/define_enum.html#wrapping-a-remote-enum